### DISCUSSION TITLE
"Automerge: enabled" but not automerging

### DISCUSSION THREAD

**Discussion by leomp12** [renovatebot/renovate#11461] [Request Help] [answered]
Hello,

I was expecting to have non-major deps grouped and automerged according to the config below:

```json
{
  "extends": [
    "config:base"
  ],
  "schedule": [
    "after 7am on the first day of the month"
  ],
  "labels": [
    "dependencies"
  ],
  "rangeStrategy": "bump",
  "packageRules": [
    {
      "matchUpdateTypes": [
        "minor",
        "patch",
        "lockFileMaintenance"
      ],
      "labels": [
        "dependencies",
        "automerge"
      ],
      "automerge": true,
      "groupName": "All non-major"
    }
  ]
}
```

They have been grouped correctly, PRs even mentions _Automerge: Enabled._, but the merge doesn't happen automatically 🤔 
Example PR here https://github.com/ecomplus/app-fb-conversions/pull/4 , I have several others in other repositories in the same way 😬 

Any idea? Thanks in advance!

**Comment by rarkins** [comment, marked as answer]
Check the logs for the PR at app.renovatebot.com and read the troubleshooting section at https://docs.renovatebot.com/automerge-configuration/

**Reply by gregoryduckworth** [reply]
I'm having similar issues but that link you provided seems to 404.

**Reply by rarkins** [reply]
https://docs.renovatebot.com/key-concepts/automerge/

**Comment by leomp12** [comment]
`"requiredStatusChecks": null` will resolve my case I guess, thanks @rarkins

**Reply by viceice** [reply]
We strongly discourage using automerge without any ci checks. This can break your target branch.

**Reply by leomp12** [reply]
Yup, I know @viceice , but I believe it's acceptable for these repositories.
Many repositories with very infrequent editions (almost maintenance free), the most important updates actually almost always involve a dependency that these repositories will have in common.
Anyway, automerge is only for non-major and if something goes wrong the deploy workflow should fail.
